src/main/java/com/coursesPlatform/trainer/TrainerController.java

```diff
@@ -34,9 +34,12 @@
         return new ResponseEntity<>(add, HttpStatus.CREATED);
     }
 
-    @PutMapping("/update")
-    ResponseEntity<TrainerDTO> updatedTrainer(@RequestBody TrainerDTO trainerDTO) throws TrainerNotFoundException {
-        TrainerDTO update = trainerService.update(trainerDTO);
+    @PutMapping("/{name}/{lastName}")
+    ResponseEntity<TrainerDTO> updatedTrainer(@PathVariable("name") String firstname,
+                                              @PathVariable("lastName") String lastname,
+                                              @RequestBody TrainerDTO trainerDTO
+    ) throws TrainerNotFoundException {
```

Comment:
unchecked exceptions (line RuntimeException) doesn't have to be declared in method definition